### PR TITLE
drop delete legacy gcroot

### PR DESCRIPTION
**Pull request by zowoq**

### CODE REVIEW

File: nix/master.nix [hunk 1]
```diff
@@ -778,11 +778,7 @@ in
       };
 
       systemd.tmpfiles.rules =
-        [
-          # delete legacy gcroot location, can be dropped after 2024-06-01
-          "R /var/lib/buildbot-worker/gcroot - - - - -"
-        ]
-        ++ lib.optional (cfg.outputsPath != null)
+        lib.optional (cfg.outputsPath != null)
```
Comment (zowoq) [inline nix/master.nix:781]: @Mic92 is `outputsPath` feature not working? It looks like it is enabled in your dotfiles but https://buildbot.thalheim.io/nix-outputs is empty?

**Comment by Mic92** [comment]
@mergify queue

File: nix/master.nix (hunk 1 above)
Comment (Mic92) [reply nix/master.nix:781]: Looks like the nginx configuration broke. I have a look. But I also just discovered a different problem. I need to introduce namespacing in there.

Comment (zowoq) [reply nix/master.nix:781]: > I need to introduce namespacing in there.

https://github.com/nix-community/buildbot-nix/pull/243

Looks like this PR handles namespacing, PRs and skipped builds.

**Pull request merged**